# Int32MultiArrayHeader.msg
# Message for handling ADCs

std_msgs/Header header
std_msgs/Int32MultiArray data
